llama_stack/providers/adapters/inference/tgi/config.py

```diff
@@ -14,6 +14,7 @@
 class TGIImplConfig(BaseModel):
     url: str = Field(
         description="The URL for the TGI endpoint (e.g. 'http://localhost:8080')",
+        default="http://localhost:8080",
```

Comment:
can you make this into host, port with a computed property like in `RemoteProviderConfig`? it is simpler and avoids mistakes.